cmd/minikube/cmd/start.go

```diff
@@ -1182,7 +1184,20 @@ func validateFlags(cmd *cobra.Command, drvName string) {
 
 	validateRegistryMirror()
 	validateInsecureRegistry()
+}
 
+// if container runtime is not docker, check that cni is not disabled
+func validateCNI(cmd *cobra.Command, runtime string) {
+	if runtime == "docker" {
+		return
+	}
+	if cmd.Flags().Changed(cniFlag) && strings.ToLower(viper.GetString(cniFlag)) == "false" {
+		if viper.GetBool(force) {
+			out.WarnReason(reason.Usage, "The '{{.name}} container runtime requires CNI", out.V{"name": runtime})
```

Comment:
We can add this message to be more clear “You have chosen to disable the CNI but ... (continue with your message)